Deposito

JonnathanE opened the pull request:
Creación de la interfaz y el ruteo para hacer depósitos y retiros.
!: falta hacer que los datos se modifiquen en la base de datos

JonnathanE committed:
Diseño para buscar cuenta cooperativa para hacer deposito terminado

JonnathanE committed:
Diseño y ruteo para realizar depósitos terminado

Falta la funcionalidad de guardar la información en la base de datos

JonnathanE committed:
Merge branch 'Developer' into deposito

JonnathanE committed:
Diseño y ruteo para realizar retiros terminado

Falta hacer la modificación en la base de datos

JonnathanE committed:
Agregar espacios en el codigo

JonnathanE committed:
Funcionalidad  nuevo socio con persistencia de datos terminado

- Persistencia de datos para crear un nuevo socio
- Se autocrea un nuevo usuario y cuenta cooperativa

JonnathanE committed:
Ver informacion del socio terminado

- Se agrego informacion de la cuenta del socio

JonnathanE committed:
Funcionalidad editar socio modificada

Se busca primero el socio y se edita

JonnathanE committed:
Funcion buscar numero de cuenta para el deposito terminado

- Se modifico el parametro @OneToOne de los modelos de usuario, socio y cuentaCooperativa
- Se realizo la busqueda de cuentaCooperativa en la base de datos por el numero de cuenta y se presenta en la vista
- Se presenta los datos en el formulario para hacer el deposito
- !: falta hacer la persistencia de datos para el deposito

JonnathanE committed:
Persistencia de datos para hacer deposito

- Se agrego el @OneToMany a los modelos de cuentaCooperativa, deposito y Socios, para hacer una relacion de doble sentido
- Los datos del deposito se guardan en la DB
- !: FALTA LA PERSISTENCIA DEL SALDO ACTUAL

JonnathanE committed:
Persistencia de datos para hacer retiros

- Se agrego el @OneToMany para los modelos Retiro y Socio, para que la relacion en JPA sea bidireccional.
- La funcionalidad de buscar cuenta y retirar con persistencia de datos terminado.
- !: FALTA LA PERSISTENCIA DE DATOS PARA EL SALGO ACTUAL

JonnathanE committed:
Actualizacion de datos del modelo usuario junto con el modelo socio

JonnathanE committed:
Funcionalidad buscar socio por cedula terminada

JonnathanE committed:
Listar cuentas cooperativa terminado

JonnathanE committed:
Funcionalidad ver informacion cuenta cooperativa terminadp

!: Hay error al redirigir desde informacion cuenta cooperativa a editar la informacion del usuario debido a que gestion usuario no usa rutas (llama directamente a la plantilla jsp)

JonnathanE committed:
Funcionalidad eliminar cuenta cooperativa terminado

JonnathanE committed:
Funcionalidad editar cuenta cooperativa terminado

JonnathanE committed:
Funcionalidad de buscar cuenta cooperativa terminado

JonnathanE committed:
Personalizar tablas

JonnathanE committed:
Funcionalidad de logueo

- se rediseño el login.
- se creo la plantilla de la pagina para el socio
- se modifico el session

JonnathanE committed:
Agregar restricciones al ingreso de las paginas

- Validar el acceso mediante el request.getSession()

joelcartuche reviewed (approved):
Aprobacion cambios cuenta socio

joelcartuche committed:
subida cambios cuenta cooperativa

suida de las configuraciones de cuenta cooperativa ademas de los gestores de retiro y credito
